Assets/Scripts/FloatingOrigin/Origin.cs

```diff
@@ -88,19 +88,20 @@ private IEnumerator AttemptShift()
         /// <param name="destination"></param>
         public void MoveOriginTo(Coordinate destination)
         {
-            Coordinate mainShifterRD = new Coordinate(mainShifter.position).Convert(CoordinateSystem.RDNAP);
-           
+            Coordinate CurrentOrigin = new Coordinate(transform.position);
```

Comment:
Dit lijkt niet te kloppen, waarom is hier mainShifter.position weggehaald en transform.position gebruikt? Ook in de code hierboven lijkt transform.position en mainShift door elkaar gebruikt te worden. Het origin object is een game object dat op 0,0,0 blijft staan terwijl de mainShifter de camera is. 

Mis ik iets?

Reply:
Okay, ik was duidelijk abuis! 

Ik heb samen met Sam hiernaar gekeken en je bent inderdaad vanaf de real world coordinaten van de origin (transform.position) aan het shiften naar de mainShifter zijn real-world coordinaten. 

Dat betekent dat dit juist een bug lijkt te fixen die er nog in zat, maar wat wel verwarring geeft is de naamgeving (sowieso: variabelen in methods in camelCase en niet in PascalCase aub), mijn suggestie zou iets van `fromCoordinates` zijn om duidelijk te maken dat dit de start van de shift is. maar wat ook niet lekker lijkt, is dat ook de mainShifterRd naamgeving in de UNIY_EDITOR blok hieronder niet de juiste naam is, dat zou -bijv- `fromCoordinatesInRd` moeten zijn

Reply:
naamgeving verbeterd